nservicebus/errors/automatic-retries.md

```diff
@@ -70,6 +70,10 @@ NOTE: Retrying messages for extended periods of time would hide failures from op
 
 SLR can be configured in several ways.
 
+### Transport transaction requirements
+
+The SLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md) and scheduling message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md) the next time it is picked from the input queue. Aborting the receive operation when transactions are turned off would results in a message loss. Therefore SLR cannot be used when transport transactions are disabled.
```

Comment:
Shouldn't this state that SLR has a dependency on FLR? FLR does the counting semantics correct?

Saying SLR cannot work because the transaction is rolled back is more an implementation issue as we are forwarding failed messages to the error queue in case of exceptions when there are no transactions.

If we are doing a best effort in moving it to the error queue then why can't we do a best effort in processing it for SLR?

It might help here to list the steps to understand the behavior.